### PR TITLE
closes #16

### LINKED ISSUE #16: Move related section to a partials
Move the section of related articles in the template to a blade partial and replace it in each respecting view

This is a screenshot of the section

image

You can find an example of the markup in [resources/views/lectures/lecture.blade.php Line 1475](https://github.com/CaddyDz/Isaac/blob/master/resources/views/lectures/lecture.blade.php#L1475) to [Line 1519](https://github.com/CaddyDz/Isaac/blob/master/resources/views/lectures/lecture.blade.php#L1519)

Put all of that markup in a file named `resources/views/partials/related_articles.blade.php` and replace that markup in `resources/views/explanations/explanation.blade.php` and `resources/views/lectures/lecture.blade.php` like so

```php
@include('partials.related_articles')
```

### PR DESCRIPTION
**Pull request by MEDANIS98**
Signed-off-by: Sadoudi Anis <sadoudimohamedanis@gmail.com>

### CODE REVIEW

File: resources/views/explanations/explanation.blade.php [hunk 1]
```diff
@@ -180,48 +180,7 @@ class="stats-add-on-click stats-add-download hvr-pulse"
...
-			<a href="https://khaledalsabt.com/explanations/982/مسلم-3-تكملة-الحديث-27-الى-الحديث-30">
-				<p class="card__title card__title--no-border card__title--font-sm">
-					مسلم 3 تكملة الحديث 27 إلى الحديث 30
-				</p>
-			</a>
-		</div>
-	</div>
-</div>
+@include('partials.related_articles')
+
```
Comment (CaddyDz) [inline resources/views/explanations/explanation.blade.php:184]: remove this line

File: resources/views/partials/related_articles.blade.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{{-- Related articles --}}
+
```
Comment (CaddyDz) [inline resources/views/partials/related_articles.blade.php:2]: remove this empty line

File: resources/views/partials/related_articles.blade.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	</h2>
+</div>
+<div class="cards cards--space-between cards--small">
+	<div class="card card--cover">
+		<div class="cover" style="background-image: url('https://khaledalsabt.com/img/types/lecture.jpg')">
+			<span class="card__gradients"></span>
+		</div>
+		<div class="card__content interpretation">
+			<a
+				href="https://khaledalsabt.com/lectures/7/%D9%88%D9%82%D9%81%D8%A7%D8%AA-%D9%85%D8%B9-%D9%82%D9%88%D9%84%D9%87-%D8%AA%D8%B9%D8%A7%D9%84%D9%89-%D9%88%D9%84%D8%A7-%D8%AA%D9%82%D9%81-%D9%85%D8%A7-%D9%84%D9%8A%D8%B3-%D9%84%D9%83-%D8%A8%D9%87-%D8%B9%D9%84%D9%85">
```
Comment (CaddyDz) [inline resources/views/partials/related_articles.blade.php:15]: remove this empty line, and shorten the URL

File: resources/views/explanations/explanation.blade.php (hunk 1 above)
Comment (MEDANIS98) [reply resources/views/explanations/explanation.blade.php:184]: done

File: resources/views/partials/related_articles.blade.php (hunk 2 above)
Comment (MEDANIS98) [reply resources/views/partials/related_articles.blade.php:2]: done

File: resources/views/partials/related_articles.blade.php (hunk 3 above)
Comment (MEDANIS98) [reply resources/views/partials/related_articles.blade.php:15]: done

**Comment by MEDANIS98** [comment]
All done

**Comment by CaddyDz** [comment]
Resolve the conversations

File: resources/views/partials/related_articles.blade.php [hunk 4]
```diff
@@ -11,10 +10,9 @@
 			<span class="card__gradients"></span>
 		</div>
 		<div class="card__content interpretation">
-			<a
-				href="https://khaledalsabt.com/lectures/7/%D9%88%D9%82%D9%81%D8%A7%D8%AA-%D9%85%D8%B9-%D9%82%D9%88%D9%84%D9%87-%D8%AA%D8%B9%D8%A7%D9%84%D9%89-%D9%88%D9%84%D8%A7-%D8%AA%D9%82%D9%81-%D9%85%D8%A7-%D9%84%D9%8A%D8%B3-%D9%84%D9%83-%D8%A8%D9%87-%D8%B9%D9%84%D9%85">
-				<p class="card__title card__title--no-border card__title--font-sm">
-					وقفات مع قوله تعالى: (ولا تقف ما ليس لك به علم)
+			<a href="https://khaledalsabt.com/lectures/7/
```
Comment (CaddyDz) [inline resources/views/partials/related_articles.blade.php:13]: Close the attribute value double quotes
```html
<a href="https://khaledalsabt.com/lectures/7/" <---- This
```

Comment (MEDANIS98) [reply resources/views/partials/related_articles.blade.php:13]: okay done

**Pull request merged**